composer/trainer/trainer.py

```diff
@@ -876,6 +959,7 @@ def __init__(
                            model=model,
                            callbacks=callbacks,
                            grad_accum=grad_accum,
+                           eval_batch_split=eval_batch_split,
```

Comment:
You probably vetted this with research team, but wouldn't we want something like `auto_eval_batch_size: bool` instead to be more discoverable a feature?

Reply:
Research preferred having grad_accum=auto affect both eval and train

Reply:
ah ok sounds good, then does `eval_batch_split` even need to be an argument to the trainer?

Reply:
I don't think so. The goal here is to add no trainer args. 

It doesn't really make sense for someone to intentionally set eval_batch_split and a high batch size (since theres no grad accum happening)

Reply:
oops, my bad I misread this code; thought it was the `__init__` signature and didn't expand the context.